src/main/java/com/google/step/snippet/external/Client.java

```diff
@@ -1,9 +1,43 @@
 package com.google.step.snippet.external;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.FilterOperator;
 import com.google.step.snippet.data.Card;
 
 public interface Client {
```

Comment:
This is no longer an interface, but an abstract class. For example, `getFeedback` is not a method you're publicizing as part of your interface, but a protected method you want subclasses to have access to.